src/apiCalls.tsx

```diff
@@ -31,3 +31,17 @@ export const postAWedding = (wedding: any) => {
     alert('Could not add wedding')
   })
 }
+
+export const getWeddingGuests = () => {
```

Comment:
Thanks for building these out! Nice to know we're all set once this endpoint is ready